docs/www/advanced-configuration.md

```diff
@@ -192,7 +192,16 @@ redpanda:
   # Default: 0.0.0.0:9092
   advertised_kafka_api:
     address: "0.0.0.0"
-    port: 9092 
+    port: 9092
+
+  # Multiple listeners are also supported as per KIP-103.
```

Comment:
I think that the same comment and example needs to be made for `kafka_api` too, right?

```
  kafka_api:
    address: "0.0.0.0"
    port: 33145
```

so the names on the endpoints have to match up.

Reply:
@BenPope  is this feedback addressed. can't tell.

Reply:
I haven't done anything yet. @dotnwat is the boss here. I can't work out why both config options support both types of config. Is there even a difference any more?

Reply:
Looks like if `kafka_api()` is the one that's used iterated on by the server, `advertised_kafka_api()` is never used, but if it's empty it returns `kafka_api()`. So I don't know.

Reply:
Oh, I found it.

Reply:
We can choose to document only the new list format. The old format is still read by redpanda, but that's a convenience for backwards compatibility.

Does that answer your question?

Reply:
Maybe we missed an opportunity to have:
```yaml
  kafka_api_multiple:
    - name: public
      listen:
        address: "127.0.0.1"
        port: 9092
      advertise:
        address: public.asdf.com
        port: 9092
    - name: private
      listen:
        address: "127.1.0.1"
        port: 9093
      advertise:
        address: internal.network
        port: 9093
```

Reply:
nice! since `kafka_api_multiple` isn't conflicting, it would be easy to add a new interface to the configuration file. but rpk doesn't mind the work either way :P

Reply:
can you create a ticket with that new format? it'd be a good improvement to make in the future